src/lib/scheduler_clients/pbs/models.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2025, ETH Zurich. All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+
+from typing import Dict, Optional
+
+from pydantic import AliasChoices, Field, validator
+
+# models
+from lib.models.base_model import CamelModel
+from lib.scheduler_clients.models import (
+    JobMetadataModel,
+    JobModel,
+    JobDescriptionModel,
+    JobTime,
+    NodeModel,
+    PartitionModel,
+    ReservationModel,
+)
+
+from datetime import datetime
+
+
+class PbsJobDescription(JobDescriptionModel):
+    name: Optional[str] = Field(default=None, description="Name for the job")
+    account: Optional[str] = Field(
+        default=None, description="Charge job resources to specified account"
+    )
+    current_working_directory: str = Field(
+        alias="working_directory", description="Job working directory"
+    )
+    standard_input: Optional[str] = Field(
+        default=None, description="Standard input file name"
+    )
+    standard_output: Optional[str] = Field(
+        default=None, description="Standard output file name"
+    )
+    standard_error: Optional[str] = Field(
+        default=None, description="Standard error file name"
+    )
+    environment: Optional[Dict[str, str]] = Field(
+        alias="env",
+        default={"F7T_version": "v2.0.0"},
+        description="Environment variables to set in the job context",
+    )
+    queue: Optional[str] = Field(default=None, description="Queue to submit to")
+    resources: Optional[str] = Field(
+        default=None,
+        description="Resource requirements (e.g. nodes, ppn, mem, walltime)",
+    )
+    script: str = Field(description="Script for the job")
+    script_path: str = Field(
+        default=None, description="Path to the job in target system"
+    )
+
+
+class PbsJobMetadata(JobMetadataModel):
+    job_id: str = Field(alias=AliasChoices("jobId", "job_id"))
+    script: None = None
+    standard_input: None = Field(
+        validation_alias=AliasChoices("StdIn", "standardInput"), default=None
+    )
+    standard_output: Optional[str] = Field(
+        validation_alias=AliasChoices("StdOut", "standardOutput"), default=None
+    )
+    standard_error: Optional[str] = Field(
+        validation_alias=AliasChoices("StdErr", "standardError"), default=None
+    )
+
+
+class JobTimePbs(JobTime):
```

Comment:
From now on, I won't repete the need to move all common fields to the generic classes (e.g. JobTime, JobModel, Node, etc.)